### ISSUE TITLE
Decrypting already downloaded dataset on HPC using vdb-decrpyt [ELI5]

### ISSUE THREAD

**Issue by MeerajKothari** [ncbi/sra-tools#362]
Hello, 

I've never done this before and I'm trying to learn this so please bear with me. My issue is similar to https://github.com/ncbi/sra-tools/issues/114 where we have a lot of data downloaded using aspera connect in a directory such as this `.../FraminghamSecure1/7XXX5/PhenoGenotypeFiles/ChildStudyConsentSet_phsXXXX2.Framingham.vXX.pXX.cX.HMB-IRB-MDS/GenotypeFiles`. 

The project key .ngc file is located in a different directory as `.../ncbi/dbGaP-1XXX3/prj_1XXX3.ngc`. I tried following the creation of a workspace instructions listed here (https://www.ncbi.nlm.nih.gov/books/NBK63512/). I created the dbGaP-1XXX3 folder manually and put the project key in it because I had it on my computer locally. The project key is the only file in this folder. 

When I call `/sratoolkit.2.10.5/bin/vdb-decrpyt .../FraminghamSecure1/7XXX5/PhenoGenotypeFiles/ChildStudyConsentSet_phsXXXX2.Framingham.vXX.pXX.cX.HMB-IRB-MDS/GenotypeFiles` I get the following error:

`2020-06-17T14:46:29 vdb-decrypt.2.10.5 err: encryption key not found while opening manager within virtual file system module - unable to obtain a password
2020-06-17T14:46:29 vdb-decrypt.2.10.5: exiting: RC(rcVFS,rcMgr,rcOpening,rcEncryptionKey,rcNotFound) (2615479768)` 

When I run vdb-config -i I get the following interface (I don't know what to do with it): 

![Screen Shot 2020-06-17 at 10 51 58 AM](https://user-images.githubusercontent.com/54863231/84913509-9f20f700-b088-11ea-9a52-2a4f66586507.png)

![Screen Shot 2020-06-17 at 10 51 53 AM](https://user-images.githubusercontent.com/54863231/84913531-a7793200-b088-11ea-98b0-872bb376b06c.png)

My issue is I don't know where to begin or how to configure the toolkit or run the decryption command. In other threads and resources it is mentioned that I don't need the .ngc key, instead I need a JWT token. What exactly is a JWT token? How do I get it and how do I use it? Also, most online guides or tutorials mention that the workspace creation happens automatically on using vdb-config -i and `prefetch`. Since the `prefetch` process has been skipped in this situation, I'm not quite able to follow the other guides. If I were to create a workspace on my own, how should I do it? 

Thank you in advance!

**Comment by TTTTaro** [comment]
Hello, have you solved this problem? I met the same issue. Could you please share your solution? Thanks a lot.

**Comment by kwrodarmer** [comment]
Please see https://github.com/ncbi/sra-tools/wiki/First-help-on-decryption-dbGaP-data .
